task/buildah/0.1/buildah.yaml

```diff
@@ -301,9 +304,6 @@ spec:
     image: $(params.BUILDER_IMAGE)
     computeResources: {}
     script: |
-      # Expose base image digests
-      buildah images --format '{{ .Name }}:{{ .Tag }}@{{ .Digest }}' | grep -v $IMAGE > $(results.BASE_IMAGES_DIGESTS.path)
-
```

Comment:
Hi Stuart, why need this movement?

Reply:
Because it needs to be run on the remote host, where the base images are actually pulled to.